proposals/lotus-light.md

```diff
@@ -0,0 +1,87 @@
+# Lotus Light Mode
+
+Authors: @magik6k
+
+Initial PR: TBD <!-- Reference the PR first proposing this document. Oooh, self-reference! -->
+
+## Purpose &amp; impact
+#### Background &amp; intent
+
+Currently, running lotus nodes requires dedicating lots of resources. Users should
+be able to talk to the blockchain without validating it fully
+
+#### Assumptions &amp; hypotheses
+* Filecoin chain can be followed just by validating blockheaders with bits of previous state safely
+* Bitswap is fast enough to support this
+
+#### User workflow example
+Instead of syncing a full node from a snapshot (big, ~90G), the client would
+start a light node, which would trust state in blockheaders, and follow the chain
+just by running basic validation on blockheaders (checking miner power/signatures
+using older state we already trust)
+
+#### Impact
+Lotus would be much easier to run for non-miner users
+
+#### Leverage
+_How much would nailing this project improve our knowledge and ability to execute future projects?_
+
+<!--
+Explain the opportunity or leverage point for our subsequent velocity/impact (e.g. by speeding up development, enabling more contributors, etc)
+-->
+
+#### Confidence
+_How sure are we that this impact would be realized? Label from [this scale](https://medium.com/@nimay/inside-product-introduction-to-feature-priority-using-ice-impact-confidence-ease-and-gist-5180434e5b15)_.
```

Comment:
```suggestion
- Medium impact: It doesn't transform the user experience, but it is at the very heart of interacting with the network (running Lotus)
- High confidence: We know that the difficulty of running Lotus is a barrier to entry for many
- Medium ease: It's a non-trivial amount of work, but it is low-risk -- the probability that we run into some insurmountable challenge is basically zero, since most obstacles will just need some refactoring to be overcome.
```